cdap-ui/app/hydrator/templates/create/toppanel.html

```diff
@@ -232,12 +232,12 @@
         </div>
       </div>
     </div>
-    <div class="btn run-time">
+    <div class="btn run-time" title="{{HydratorPlusPlusTopPanelCtrl.queueStatus}}">
```

Comment:
Is this intended to be `"{{HydratorPlusPlusTopPanelCtrl.queueStatus}}"`? Could it be perhaps `{{HydratorPlusPlusTopPanelCtrl.queueStatus}}` instead?

Reply:
Since the expression is being used for an attribute, we need double quotes around it.

Reply:
Ah I see! Thanks for clarifying it :)